test/scripts/boot-image

```diff
@@ -66,18 +78,34 @@ def ensure_uncompressed(filepath):
 def cmd_boot_aws(arch, image_name, privkey, pubkey, image_path, script_cmd):
     # pylint: disable=too-many-arguments
     aws_config = get_aws_config()
-    cmd = ["go", "run", "./cmd/boot-aws", "run",
-           "--access-key-id", aws_config["key_id"],
```

Comment:
Uh, let's please not do this change here, this makes the `cmd` much harder to read IMHO. It's a bit silly that `ruff` cannot be configured to do better here (or can it)?

Reply:
I saw that too - I'll switch this to draft and dig deeper